project/src/main/puzzle/puzzle-score.gd

```diff
@@ -276,12 +276,13 @@ func get_creature_score() -> int:
 	return creature_scores[creature_scores.size() - 1]
 
 
-func get_creature_line_clears() -> int:
-	return creature_line_clears[creature_line_clears.size() - 1]
-
-
-func set_creature_line_clears(value: int) -> void:
-	creature_line_clears[creature_line_clears.size() - 1] = value
+func set_combo(value: int) -> void:
```

Comment:
Our guidelines specify that setters follow the convention `set_foo(new_foo: Type) -> void`

Reply:
Changed to `func set_combo(new_combo: int) -> void:`